### PR TITLE
Re-land "[Windows] fixing window glitches while moving or resizing" 

### LINKED ISSUE #11777: Window glitches when dragging app between monitors with different scaling
### Description

Dragging a Maui app between different monitor causes visual glitches and eventually stops responding. See attached video to see what I mean: [screen-capture.webm](https://user-images.githubusercontent.com/62772237/204944972-ed0e29f2-b5b8-465c-905a-96af43d966f7.webm). In the video I'm just recording my 2nd monitor; the laptop screen is below it so that's where the app is when I drag it off screen below. In the end I need to ctrl+alt+delete to get to the task manager and force close it because everything stops responding.

### Steps to Reproduce

1. Have a dual-monitor setup
2. Set custom scaling on monitor in the display settings. e.g. I have my laptop scaling set to 125% and a secondary monitor with scaling at 100%. (If I change the laptop to also be 100% scaling then the bug does not occur).
image
4. Create a new app from the .Net MAUI App template (using .Net7)
5. Launch the app
6. Drag the app from one monitor to the other. If it's still behaving normally, drag it back.

Expected results:
You can freely move the app around and it will work fine.

Actual results:
After moving from one monitor to the other a few times, it will break. Sometimes the app window flashes around wildly, sometimes just the edge of the window twitches - see video in description.

### Link to public reproduction project repository

Can repro in the Maui template app

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 10 19043.2251

### Did you find any workaround?

Don't use different scaling

### Relevant log output

_No response_

### LINKED ISSUE #14597: Moving MAUI Window between monitors with different scalling has issues
### Description

Moving MAUI window between monitors that have different scaling causes some weird behaviour like window shifting to an unexpected location or moving away from cursors original position and etc..

Here is a video of the behaviour :

https://user-images.githubusercontent.com/1272095/232254330-3c593ced-a8bf-4c62-a848-f6f16a73c3f7.mp4

### Steps to Reproduce

1. Have two monitors
2. Set one monitor to 100% scaling and the other one to 150% scaling
3. Try to move MAUI window slowly between monitors
4. Now try moving the window faster between monitors. 

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows

### Did you find any workaround?

No

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change

Initially:
- #14861

Reverted:
- #16541

### Issues Fixed

Fixes:

- Fixes #11777
- Fixes #14597

### CODE REVIEW

**Comment by salarcode** [comment]
Thanks for adding this back.
Currently with `SetValueCore` not respecting `SetValuePrivateFlags.Silent` value, this fix doesn't work.
More info https://github.com/dotnet/maui/pull/16541#issuecomment-1671284416

**Comment by mattleibow** [comment]
@StephaneDelcroix is there an alternative api that should be used?

**Comment by StephaneDelcroix** [comment]
I'm currently looking at this

**Review by rmarinho — DISMISSED** [review]

File: src/Controls/src/Core/BindableObject.cs [hunk 1]
```diff
@@ -440,16 +440,17 @@ internal void SetValueCore(BindableProperty property, object value, SetValueFlag
 			}
 			else
 			{
+				var silent = (privateAttributes & SetValuePrivateFlags.Silent) != 0;
 				context.Attributes |= BindableContextAttributes.IsBeingSet;
-				SetValueActual(property, context, value, currentlyApplying, attributes, specificity);
+				SetValueActual(property, context, value, currentlyApplying, attributes, specificity, silent);
```
Comment (rmarinho) [inline src/Controls/src/Core/BindableObject.cs:445]: Should we add a unit test for this?

**Review by samhouts — DISMISSED** [review]

File: src/Controls/src/Core/BindableObject.cs (hunk 1 above)
Comment (mattleibow) [reply src/Controls/src/Core/BindableObject.cs:449]: Added here: https://github.com/dotnet/maui/pull/16676

**Review by jfversluis — APPROVED** [review]

**Pull request merged**